charts/argo-cd/Chart.yaml

```diff
@@ -22,4 +23,8 @@ dependencies:
     condition: redis-ha.enabled
 annotations:
   artifacthub.io/changes: |
-    - "[Fixed]: Type conversion for ConfigMap values"
+    - "[Security]: Use recommended container security contexts by default"
+    - "[Added]: Container security context for server UI extensions sidecar"
+    - "[Fixed]: Redis metrics sidecar now uses correct configuration option"
+    - "[Removed]: ApplicationSet securityContext in favor of global.securityContext"
```

Comment:
Why do we want to remove the ability to customize pod level SecurityContext per component? Maybe some addons require special privileges, idk?